README.md

```diff
@@ -10,19 +10,7 @@ Employees of Heroku see: [The Ruby language guides](https://github.com/heroku/la
 
 ## How it works
 
-Logic lives in the `build.rb` script at the root of this project. It will call `./configure` and `make` with the corresponding inputs. This file is coppied into a docker image when `$ bundle exec rake "generate_image[heroku-22]"` is called. See `dockerfiles/Dockerfile.heroku-22` for an example.
-
-You can see the rake code:
-
-```ruby
-desc "Build docker image for stack"
-task :generate_image, [:stack] do |t, args|
-  require 'fileutils'
-  FileUtils.cp("dockerfiles/Dockerfile.#{args[:stack]}", "Dockerfile")
-  system("docker build -t hone/ruby-builder:#{args[:stack]} .")
-  FileUtils.rm("Dockerfile")
-end
-```
+Logic lives in the `build.rb` script at the root of this project. It will call `./configure` and `make` with the corresponding inputs. This file is coppied into a docker image when `$ bin/build_ruby` is called. See `dockerfiles/Dockerfile.heroku-24` for an example.
```

Comment:
```suggestion
Logic lives in the `build.rb` script at the root of this project. It will call `./configure` and `make` with the corresponding inputs. This file is copied into a docker image when `$ bin/build_ruby` is called. See `dockerfiles/Dockerfile.heroku-24` for an example.
```